src/public/about.html

```diff
@@ -31,20 +30,18 @@
 
       <div class="main-width">
         <main>
-          <h2>
```

Comment:
Let's add `<h2>About</h2>` here, just inside the `<main>` tag, so people quickly see which page they are on.